packages/portalnetwork/src/subprotocols/beaconChain/types.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ByteListType,
+  ByteVectorType,
+  ContainerType,
+  ListCompositeType,
+  UintBigintType,
+  UnionType,
+  BitVectorType,
+} from '@chainsafe/ssz'
+
+// todo: export
+
+/* ----------------- Alliases ----------------- */
+const Bytes4 = new ByteVectorType(4)
+const Bytes32 = new ByteVectorType(32)
+const Bytes48 = new ByteVectorType(48)
+const Bytes96 = new ByteVectorType(96)
+/* ----------------- Types ----------------- */
+
+const Slot = new UintBigintType(64)
```

Comment:
Typically we'll want to also define Typescript types for each SSZ type alias.  I'm open to suggestions on naming conventions for this, but it looks something like:

`type Slot = bigint`
`const SlotType = new UintBigIntType(8)`

So that our functions can look like:
`function (s: Slot) => { ... SlotType.serialize(s) ...}`